### ISSUE TITLE
Error how can i resolve it

### ISSUE THREAD

**Issue by rimsha234** [spmallick/learnopencv#902]
8 | #include <TH/TH.h>
      |          ^~~~~~~~~
compilation terminated.
error: command '/usr/bin/g++' failed with exit code while running this !./make.sh

**Comment by rimsha234** [comment]
https://learnopencv.com/object-tracking-and-reidentification-with-fairmot/
i am trying to run this project and i am encountering error due to this
command !./make.sh
I am running it on colab i think this due to old version of pytorch can i
resolve this error or is this code available in newer version

>
>
>

**Comment by rimsha234** [comment]
Yes i am running this code no this line doesnt exectued .updated version is
downloaded not this one

**Comment by rimsha234** [comment]
Please have a look

**Comment by rimsha234** [comment]
There is error when i try to install the version of torch is there any code
available in new version?

**Comment by ajax619** [comment]
Any solution as i am encountering same problem

**Comment by rimsha234** [comment]
No i didnt get any solution

**Comment by rimsha234** [comment]
is there any updated code available with new dependecies will you help me
this ?

**Comment by Tech3467** [comment]
> > Yes i am running this code no this line doesnt exectued .updated version is downloaded not this one
> > […](#)
> 
> Can you provide more details, please?
> 
> Which line do you mean, can you split the multiple lines into separate lines and execute each line separately - and then show the results you are getting?

hello can you please share your email

**Comment by Tech3467** [comment]
<img width="706" alt="Screenshot 2024-05-31 102230" src="https://github.com/spmallick/learnopencv/assets/163503071/94616a80-451f-474e-b5da-22eaa2da20a7">
<img width="514" alt="2" src="https://github.com/spmallick/learnopencv/assets/163503071/769665b1-dd89-4ec1-b35a-f2cdb8f53df3">
i am encountring this error at the last line

**Comment by Hemilibeatriz** [comment]
> # Clone and install DCNv2.
> %cd /content/FairMOT
> %rm -rf /content/FairMOT/DCNv2
> !git clone -b pytorch_2.0.1 https://github.com/lbin/DCNv2
> %cd /content/FairMOT/DCNv2
> !chmod +x ./make.sh
> !./make.sh
> 
> import sys
> sys.path.append('/content/FairMOT/DCNv2')

These steps with the changes mentioned previously (remove the version of torch and torchvision, comment the line #!gdown --fuzzy https://drive.google.com/file/d/1pl_-ael8wERdUREEnaIfqOV_VF2bEVRT/view and use a different repository and a different branch name) made it work perfectly *-*.. is it possible to make its output be a video file instead of several frames?

**Comment by Hemilibeatriz** [comment]
> > Estas etapas com as alterações mencionadas anteriormente (remova a versão do torch e torchvision, comente a linha #!gdown --fuzzy https://drive.google.com/file/d/1pl_-ael8wERdUREEnaIfqOV_VF2bEVRT/view e use um repositório diferente e um nome de branch diferente) fez com que funcionasse perfeitamente ... é possível fazer com que sua saída seja um arquivo de vídeo em vez de vários quadros?
> 
> Sim, claro, a saída também pode ser um vídeo.
> 
> Dê uma olhada no método `def demo(opt):`(célula 7?) - adicione `opt.output_format = 'video'`à próxima célula (célula 8?) Com a matriz de opções de argumento.
> 
> A última seção (última frase no final do caderno) menciona:
> 
> > Confira o arquivo [opts.py](https://github.com/ifzhang/FairMOT/blob/master/src/lib/opts.py) para mais argumentos.
> 
> => existem muitas opções de mopre possíveis, consulte " https://github.com/ifzhang/FairMOT/blob/master/src/lib/opts.py ".

Thank you very much for the clarifications! In the end it actually generated a video... thank you. I needed to load two videos for it, and concatenate the two videos to simulate a single video. Do you know if it supports two inputs? I looked at the options and it doesn't mention anything... without wanting to take advantage of your good will too, do you know if this way is the most appropriate for doing multi-camera tracking? I'm trying to get a good tracker for this function, if you also know of a model that addresses this function... it would be of great help

**Comment by Tech3467** [comment]
Is this project able to reidentify the individual comes after some seconds
in the frame?

**Comment by Hemilibeatriz** [comment]
This Colab was working perfectly until last Friday... when I ran the code on Monday, I started getting an error while downloading the file... I thought it was a site instability (I tried accessing the link through the browser, but also without success), but I keep getting the same error. Any idea how to "fix" this?

![image](https://github.com/spmallick/learnopencv/assets/62450495/b16c9acc-6f10-4192-803c-6fa8360b3755)
![image](https://github.com/spmallick/learnopencv/assets/62450495/607ca648-4929-4441-8aad-764db2667933)

**Comment by Hemilibeatriz** [comment]
How can i use yolov5 instead dla-34? I changed the 'load_model' and deleted 'arch' but doesn´t work.

**Comment by Hemilibeatriz** [comment]
Yes, i´ve tried others ways but i want to change the fairmot´s model :) 

Yes, i want use the yolov5 model instead dla-34, your repo says be possible, but when made the changes, it doesn't work.
